res/kime-xdg-autostart

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+/usr/bin/kime "#@"
+if systemctl --user is-active --quiet app-kime@autostart.service; then
```

Comment:
Is this line failed when systemctl command is not exists?

Reply:
yes, tested on an openrc environment.